eventually/utils/validators.py

```diff
@@ -98,11 +98,10 @@ def required_keys_validator(data, keys_required, strict=True):
 
     :return: `True` if data is valid and `False` if it is not valid.
```

Comment:
keys_required param = list 
change it in docs to set

Reply:
okey